python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -256,7 +256,8 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
 
         context = psfex.Context(prefs.getContextName(), prefs.getContextGroup(),
                                 prefs.getGroupDeg(),
-                                psfex.Context.REMOVEHIDDEN if False else psfex.Context.KEEPHIDDEN)
+                                bool(psfex.Context.REMOVEHIDDEN if False
+    				else psfex.Context.KEEPHIDDEN))
```

Comment:
Two comments:
* I think the original was more readable (when everything was on a single line). It's good practice to keep the ternary operator `if` always all on one line; use a regular `if` if it can't fit.
* Is this a manual debugging switch (`if False` will always eval to False, of course)?